src/probnum/diffeq/odefiltsmooth/odefiltsmooth.py

```diff
@@ -209,41 +201,37 @@ def probsolve_ivp(
     >>> solution = probsolve_ivp(ivp, method="eks1", which_prior="ioup3", step=0.1)
     >>> print(solution.y.mean)
     [[0.15      ]
-     [0.20837029]
-     [0.28203135]
-     [0.36939685]
-     [0.46639109]
-     [0.56591654]
-     [0.66046815]
-     [0.74369078]
-     [0.81236993]
-     [0.86592196]
-     [0.90595777]
-     [0.93495254]
-     [0.95543867]
-     [0.96968542]
-     [0.9794734 ]
-     [0.98614926]]
+     [0.20845921]
+     [0.28195738]
+     [0.36946571]
+     [0.46638028]
+     [0.56596141]
+     [0.66045798]
+     [0.74373419]
+     [0.81236692]
+     [0.86592532]
+     [0.90598047]
+     [0.93495402]
+     [0.9554465 ]
+     [0.96968678]
+     [0.97948238]
+     [0.98614302]]
 
     """
     gfilt, firststep, stprl = _create_solver_inputs(
-        ivp, method, which_prior, tol, step, firststep, precond_step, **kwargs
+        ivp, method, which_prior, tol, step, firststep, **kwargs
     )
     with_smoothing = method[-2] == "s" or method[-1] == "s"
     solver = GaussianIVPFilter(ivp, gfilt, with_smoothing=with_smoothing)
-    solution = solver.solve(firststep=firststep, steprule=stprl, **kwargs)
+    solution = solver.solve(firststep=firststep, steprule=stprl)
     return solution
 
 
-def _create_solver_inputs(
-    ivp, method, which_prior, tol, step, firststep, precond_step, **kwargs
-):
+def _create_solver_inputs(ivp, method, which_prior, tol, step, firststep, **kwargs):
```

Comment:
these kinds of functions need a serious update soon, now that much of the APi has changed. I guess the number of lines can be cut in half